articles/connections/social/google.md

```diff
@@ -10,56 +10,47 @@ alias:
  - google-oauth2
 seo_alias: google
 ---
-# Connect your app to Google
-
-To connect your Auth0 client to Google, you will need to generate a *Client ID* and *Client Secret* in a Google project, copy these keys into your Auth0 settings, and enable the Connection.
+# Connect Your App to Google
 
 ::: note
-This doc refers to the client steps to connect your client to Google. If you are looking to manage authentication in your app, see [Next Steps](#next-steps) below.
+This doc only shows you how to connect your client to Google. To manage your app's authentication, see [Next Steps](#next-steps) for links to the appropriate docs.
 :::
 
-## Generate the Google Client ID and Client Secret
-
-1. While logged in to your Google account, go to the [API Manager](https://console.developers.google.com/projectselector/apis/credentials).
-
-2. Create your new app by navigating to **Credentials** using the left-hand menu:
-
-  ![API Manager Credentials](/media/articles/connections/social/google/credentials.png)
-
-3. While you are on the **Credentials** page, click on **Create a project**.
+To connect your Auth0 client to Google, you will need to:
 
-4. In the dialog box that appears, provide a Project name, answer Google's email- and privacy-related questions, and click **Create**:
+* Generate a *Client ID* and *Client Secret* in a Google project
+* Copy these keys into your Auth0 settings
+* Enable the Connection
 
-  ![Create New Project](/media/articles/connections/social/google/create-new-project.png)
-
-5. Google will take a moment to create your project. When the process completes, Google will prompt you to create the credentials you need.
+::: warning
+Google OAuth clients requesting sensitive OAuth scopes may be [subject to review](https://developers.google.com/apps-script/guides/client-verification) by Google.
+:::
 
-  ![Create Google Credentials](/media/articles/connections/social/google/create-credentials.png)
+## Generate the Google Client ID and Client Secret
```

Comment:
I suggest adding numbers to the titles, to denote steps, like we do here https://auth0.com/docs/connections/social/facebook